pyEpiabm/docs/source/core.rst

```diff
@@ -0,0 +1,29 @@
+****
+Core
+****
+
+.. currentmodule:: pyEpiabm
+
+...
```

Comment:
Looks good, thanks for making all the changes

But by the "..." I meant replace that seirmo sentence with an appropriate one for epiabm !

Reply:
My bad -  I thought this was a syntax thing! Now corrected